src/style.css

```diff
@@ -1 +1,60 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-bars {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir, sans-serif;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.search-bars:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.search-bars:focus {
+  background-image: url(images/Search.svg),
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+  border-radius: 3px;
```

Comment:
You've changed the `border-radius` in the `:focus` state. This can lead to a noticeable jump in the UI when the input is focused. It's recommended to keep the `border-radius` consistent across different states for a smoother user experience.

Reply:
I followed the structure defined in the mockup.